core/templates/dev/head/components/forms/custom-forms-directives/object-editor.directive.ts

```diff
@@ -0,0 +1,57 @@
+// Copyright 2014 The Oppia Authors. All Rights Reserved.
```

Comment:
I see an addition here but no deletion?

Reply:
Deleted the file. It somehow didn't move but copied.
Done!